src/test/get_pipeline_metadata_response_test.cpp

```diff
@@ -222,14 +220,14 @@ TEST_F(GetPipelineMetadataResponseBuild, HasCorrectShape) {
         {}));
 }
 
-TEST_F(GetPipelineMetadataResponse, ModelVersionNotLoadedAnymore) {
+TEST_F(GetPipelineMetadataResponse, ModelVersionNotLoadedAnymoreButPipelineNotReloadedYet) {
     pipelineDefinition.mockStatus(StatusCode::MODEL_VERSION_NOT_LOADED_ANYMORE);
-    EXPECT_EQ(ovms::GetModelMetadataImpl::buildResponse(pipelineDefinition, &response, manager), ovms::StatusCode::MODEL_VERSION_NOT_LOADED_ANYMORE);
+    EXPECT_EQ(ovms::GetModelMetadataImpl::buildResponse(pipelineDefinition, &response, manager), ovms::StatusCode::OK);
```

Comment:
Why do we change this behavior?

Reply:
If PipelineDefinition is loaded then the inputs/outputs are saved and there is no need to ask ModelInstance for inputs/outputs. If you don't have PipelineDefinitionGuard then you should not ask for inputs/outputs info.